src/OrleansProviders/Streams/Memory/IMemoryStreamQueueGrain.cs

```diff
@@ -1,26 +1,22 @@
-﻿using System.Collections.Generic;
+﻿
+using System.Collections.Generic;
 using System.Threading.Tasks;
 
-namespace Orleans.Providers.Streams.Memory
+namespace Orleans.Providers
```

Comment:
Generally we try to keep namespaces short and simple in Orleans core assemblies.  Usually 'orleans' or 'oreans.runtime'.  Since this is in providers I felt Orleans.providers was appropriate.  The long namespaces in generator was my mistake.   I considered changing it with this PR but felt it was orthogonal, though I do think we should change the generators namespaces as well.